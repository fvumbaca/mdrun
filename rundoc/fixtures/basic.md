# Header 1
Some content
```sh
echo "Hello world"
```
