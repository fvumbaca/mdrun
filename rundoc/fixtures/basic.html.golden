<!DOCTYPE html>
<html>
<head>
  <title></title>
  <meta name="GENERATOR" content="Blackfriday Markdown Processor v2.0">
  <meta charset="utf-8">
</head>
<body>

<script src='/-/static/js/app.js'></script>
<h1>Header 1</h1>

<p>Some content</p>

<pre><code class="language-sh">echo &quot;Hello world&quot;
</code></pre>
<div id="c2hlY2hv"><button onclick="execBlock('c2hlY2hv')">Run</button></div>
</body>
</html>
